### ISSUE TITLE
Project 1: Restoration Activities/Ecosystem Services Synthesis Project

### ISSUE THREAD

**Issue by kdorans** [NCEAS/oss-2017#36]
Author: **Author 1**, **Author 2**, **Author 3**, ...
Topics: list of topics, comma separated

**Project Overview**

Title: Developing a framework to assess restoration success in the Gulf using prior knowledge 

Issue: Success of restoration projects in the Gulf of Mexico is not systematically characterized.

Research Questions:
How have restoration projects to-date influenced water quality in the Gulf of Mexico?
Can we use this information to inform future restoration projects aimed at improving water quality?
How much prior knowledge is needed to inform future restoration projects?

Deliverables:
The project will provide information about the change in water quality in these 3 areas pre- and post- restoration activities.
We will develop a Bayesian framework for assessing restoration projects related to water quality using knowledge gained from prior restoration activities.
We may apply the model to approved restoration activities to estimate restoration outcomes for planned or potential restoration activities. 

General approach
We will focus on three watersheds in Texas, Florida, Louisiana (?)
We will carry out a systematic review of available literature to evaluate the known relationships between ecosystem restoration projects and changes to water quality in the different watersheds.
We will develop a Bayesian Decision Network that will quantify and test the relationships between decision nodes (restoration activities), indicators of water quality, and watershed water quality improvement.
We may also further expand this model to assess impacts on demographic indicators. 

**Additional Details of Project to Consider for Further Development**

A. Summary of Synthesis

Significant investments are currently being made to understand and restore water quality throughout the Gulf Coast. Building upon the lessons learned from previous restoration activities in the Gulf, we aim to develop a decision support tool which will guide future restoration activities within the Gulf. Our goal is to synthesize the available information on Gulf Coast environmental and social quality to better direct research and restoration activities toward the coast's continued improvement.  

Specific Research Questions:
Can hydrological data be used to estimate restoration outcomes?
Are there sufficient data at a watershed scale to address assess restoration outcomes?
How does restoration success vary among watersheds? (Social and ecological function)
What are the data we would need to improve restoration decision making?

Conceptual Model: Restoration --> Ecological Function --> Outcomes

Test using:
Bottom-up analysis: 5 water quality measures for 3 watersheds to generate an estimate of efficacy of restoration activities.
1-2 social metrics (Census data)

Watersheds/Restoration Activities: TB):
Texas (salinity), Louisiana (sediment), FL (nutrients)

B. Data needs
B.1. Environmental Quality
B.1.1. Estuarine/near coastal Water Quality: 
[Tampa Bay](http://www.tampabay.wateratlas.usf.edu/datadownload/Default.aspx), 
[Mobile Bay](http://cf.disl.org/mondata/disclaimer.cfm), 
[NERRS datasets](http://cdmo.baruch.sc.edu/) for Weeks Bay (AL), Apalachicola Bay (FL), Rookery Bay (FL), Grand Bay (MS), Mission Aransasas (TX)
[Gulf of Mexico - Near Coastal](http://data.gcoos.org/) 
[Louisiana Coastwide Reference Monitoring System](https://cims.coastal.louisiana.gov/monitoring-data/default.aspx)

B.1.2. Coastal Environment
[NLCD 2011](https://www.mrlc.gov/nlcd2011.php)
[Stream/River inputs to coastal areas from, USGS gages](https://waterdata.usgs.gov/nwis/dv/referred_module=sw), obtained from R [dataRetrieval](https://github.com/USGS-R/dataRetrieval) package.
[Sea Level Rise Trends](https://tidesandcurrents.noaa.gov/sltrends/sltrends.html)
[Coastal Relief/Elevation](https://www.ngdc.noaa.gov/mgg/coastal/crm.html)
[Ecosystem Services](http://gomaportal.tamucc.edu/gecoview/)
[EPA EnviroAtlas](https://www.epa.gov/EnviroAtlas), also ecosystem services
[Nutrient Inputs](https://www.ncddc.noaa.gov/website/DataAtlas/atlas.htm?plate=Water%20Quality%20-%20Nutrients)

B.1.3. Coastal Biota
[Essential Fish Habitat & Life History](http://portal.gulfcouncil.org/EFHreview.html)
[Migratory Bird Hotspots](http://ebird.org/ebird/explore) 
[Species of Special Concern Essential Habitats](https://marinecadastre.gov/nationalviewer/)

B.2. Human Dimension/Social Quality
B.2.1. Demographic/Social Characteristics
County level census data, 5 or 10 year intervals depending on source from R, [tidycensus](https://github.com/walkerke/tidycensus) package. [Additional tutorials](https://juliasilge.com/blog/using-tidycensus).
[Social Vulnerability Indices](https://coast.noaa.gov/dataregistry/search/collection/info/sovi)
[Critical Facilities](https://coast.noaa.gov/dataregistry/search/dataset/info/criticalfacilities)
[Coastal Economic Impact](https://coast.noaa.gov/dataregistry/search/collection/info/coastaleconomy)
[County-Level Coastal Resource Assessments](https://coast.noaa.gov/snapshots/) 
[Coastal Resilience/Risk Summaries](http://maps.coastalresilience.org/gulfmex/)
[Shipping Industry/Vessel Traffic Summaries](https://marinecadastre.gov/nationalviewer/)

B.2.2. Coastal Impacts
[NRDA Damage Assessments](https://gomex.erma.noaa.gov/erma.html#/x=-94.28410&y=29.22159&z=5&layers=16+6770+15879+19872+19897)
[Future Urbanization](https://cida.usgs.gov/thredds/catalog.html?dataset=cida.usgs.gov/serap/serap_urb)
[Future Habitat Vulnerability](https://www.sciencebase.gov/catalog/item/55f742a8e4b0477df11c0a2b)

C. Analytical approaches
Available data resources across the Gulf will be summarized at appropriate spatial scales according to environmental and social quality indicators/metrics. Simple empirical relationships, as well as, more flexible bayesian decision networks will be developed that partition the environmental and social characteristic data along the Gulf Coast into logical, priority groupings for future research and/or restoration activities. For example, the priority groupings could provide a spatial assessment framework for identifying areas where informational data gaps exist (e.g. additional research needed in the form of water quality, habitat, biota, social data collection efforts) or where disparate restoration activities should be focused (e.g. water quality restoration vs. coastal habitat restoration vs. coastal community support & incentives). If robust models are developed, then the group could explore how discrete management decisions/scenarios may improve the environmental and social quality metrics utilized in model development.

D. Impacts
Ultimately, this group synthesis project will advance the Gulf Research Program’s goal of understanding how the human and environmental spheres interact in the Gulf of Mexico. An assessment tool/framework that helps guide future research and restoration activities towards enhancing both the coastal resource and community quality of the region will be a direct outcome.

## Original projects
Projects #34, #32, #28 and possibly #11, #22 (along with perhaps some others) might be able to be merged into a project that focuses on synthesizing information that could help guide future restoration activities. 

#34 (Synthesizing ecological, social, economic and cultural data for assessment of ecosystem services)
#32 (Developing a framework for robust research, monitoring and restoration activities)
#28 (A synthesis of multiple datasets describing water quality of coastal regions in the Gulf of Mexico)
#11 (Impacts of water level fluctuations, land area change, and demographic change in the Gulf of Mexico)
#22 (Diagnostic analysis of Gulf area using Bayesian Decision Networks)

(Proposals submitted by : @JessicaHenkel,  @esherwoo77, @fawda123, @askolker, @pvarelag).

**Comment by fawda123** [comment]
Good call, I had a similar grouping in my response to #11.  I'd be okay with this but don't have much experience with Bayesian Decision Networks.  Not opposed to it but I have yet to see an easy-to-use, open source platform.  Maybe @pvarelag has some ideas?

**Comment by pvarelag** [comment]
Yes! I have used a software called GeNIe that allows you to draw Bayesian networks, define probabilities and update models. It's very straight forward, and can be downloaded for free for academic purposes from this site:
https://download.bayesfusion.com/files.html?category=Academia

**Comment by fawda123** [comment]
Here are some useful datasets to consider from the water quality side, mostly coastal (estuarine) and inflow data.  These are all public-facing and easily accessible.

Tampa Bay: http://www.tampabay.wateratlas.usf.edu/datadownload/Default.aspx
Mobile Bay: http://cf.disl.org/mondata/disclaimer.cfm
Five smaller Gulf Coast estuaries (Weeks Bay - AL, Apalachicola Bay - FL, Rookery Bay - FL, Grand Bay - MS, Mission Aransasas - TX): http://cdmo.baruch.sc.edu/

These datasets cover several decades for common wq parameters, e.g., temp, DO, salinity, P, N, etc. I have most of them already in a usable format as RData files.

Input conditions from USGS stream gages could also be used to supplement the data: https://waterdata.usgs.gov/nwis/dv/?referred_module=sw. These are easily retrieved with the R [dataRetrieval](https://cran.r-project.org/web/packages/dataRetrieval/index.html) package.

I've also got wq data for several GOM transect surveys but I'll have to check if this okay to share with the workshop. 

Combining these datasets could provide a general overview of changing wq conditions in the Gulf from landscape to ocean, with the idea that we'd be relating wq to other datasets, e.g., demographic,, economic, etc.  I could tackle the wq component for this proposal, depending on how it fleshes out.

**Comment by esherwoo77** [comment]
I've been tied up with conference calls for another project this week, but will give some thought to adding more information to the project description above. In the mean time, there is a recent pub and data available that describes the distribution and future vulnerability of coastal habitats throughout the US GOM coast. This may be a useful dataset for the synthesis concept outlined, thus far. More info here: https://pubs.er.usgs.gov/publication/70175253  .

**Comment by fawda123** [comment]
Just came across this blog today about retrieving census data in R: https://juliasilge.com/blog/using-tidycensus/

It seems straightforward to get demographic, county-level data over time with this package.  Linking changes in Gulf county demographics with other variables (water quality, SLR, ecosystem services) could go a long way.  

I've started populating some of the data needs in the proposal.  It would be worth discussing objectives/goals of the analysis.

**Comment by vdtobias** [comment]
I ran across a Dept. of Interior website about restoration after the Deepwater Horizon spill: https://www.doi.gov/deepwaterhorizon
I'm not sure if there is anything useful there, but I thought it would be good to bookmark it.

**Comment by esherwoo77** [comment]
Hi all (@JessicaHenkel, @esherwoo77, @fawda123, @askolker, @pvarelag, @vdtobias ),
I took a shot at synthesizing our proposals into a group project concept (see above). This is a really rough outline and synthesis, so please add, edit, delete as you see fit. If the direction/concept is totally off, please feel free to revise accordingly, as well. Looking forward to meeting everyone in California soon!

**Comment by vdtobias** [comment]
I think this proposal might be headed towards significant overlap with Project 2 (#35).  Maybe there is an opportunity to create some synergies by sharing common information (like data on disturbance), but making a more clear distinction between the two projects.  Maybe this project could focus on the ecological aspects of disturbance and restoration while Project 2 could focus on the social and economic aspects.  Then the papers could complement each other.

**Comment by vdtobias** [comment]
Maybe also consider some of the ideas in #31 for this proposal.  (That's mine, for the record.)  It sounds like the Bayesian Decision Networks might be similar to the state-space models that I proposed in terms of their ability to take in data and make predictions about conditions.  @pvarelag can correct me if I'm off base here.  I've never actually fit the kind of model that I proposed so I'm not advocating for one over the other.  I'm just thinking that the framework of looking at transitions from one state to another might be useful for looking at the interactions between disturbance, ecological change, and restoration practices/effectiveness.  Whatever framework is used to look at those changes, I like where Project 1 is headed.

**Comment by JessicaHenkel** [comment]
I'm wondering if there would be a benefit to looking at existing restoration plans/activities/projects and tying them to some of the social datasets so we could make predictions about the social benefits of those restoration activities? If so, the [DWH Project Tracker](http://www.dwhprojecttracker.org/download-project-information/) has some very useful data that can be downloaded.

**Comment by kdorans** [comment]
@JessicaHenkel, that's a great idea! I also wanted to pass on this link, which shows how scientists are using the [TNC Coastal Resilience tool for GOM restoration](https://www.nature.org/ourinitiatives/regions/northamerica/areas/gulfofmexico/gom-dss.xml) (tool linked Coastal Resilience/Risk Summaries in main text above).

**Comment by pvarelag** [comment]
Here is some info that might overlap with some of the previously shared data sources:
**Social Vulnerability Index** (datasets and documentation): https://svi.cdc.gov/SVIDataToolsDownload.html
**High Resolution Bathymetry GoM:** https://www.boem.gov/Gulf-of-Mexico-Deepwater-Bathymetry/
**Permits for exploration and development of oil and gas in GoM:** https://www.boem.gov/GOMR-GIS-Data-and-Maps/
**Other references:** http://www.arcgis.com/home/search.html?q=Gulf%20of%20Mexico&start=1&sortOrder=desc&sortField=relevance
Since land cover changes is also an important marker for some of the impacts we're trying to assess here, I suggest the review of remote sensing data from sources such as:
http://www.opentopography.org/
https://earthexplorer.usgs.gov/

**Comment by ailich** [comment]
In case this is useful, there was a paper published in 2013 that looked at the ecosystem services of different habitats in the Gulf of Mexico

http://www.tandfonline.com/doi/abs/10.1080/21513732.2013.811701

**Comment by pvarelag** [comment]
Here's an article about an application of BNs for water quality assessment.
http://www.sciencedirect.com/science/article/pii/S2212613916300976

**Comment by vdtobias** [comment]
Something to consider re: Sediment for restoration in Louisiana: at least for salt marshes, restoring sediment isn't the only issue for Louisiana. Plants are sometimes the controlling factor. At least check out the abstract of this paper:
http://www.rnr.lsu.edu/people/nyman/pubs/nymECSS2006.pdf
I have some additional resources, if you're interested.

**Comment by kdorans** [comment]
Notes file: https://docs.google.com/document/d/1fi9dXCcGI-arCppHqKbcHNAjh_7T_lAmTopENVz8xQ0/edit?usp=sharing

**Comment by pvarelag** [comment]
Here's another article with BNs for restoration activities 
http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2427.2009.02219.x/full

**Comment by JessicaHenkel** [comment]
Added text developed today to the top of the file under 'Project Overview' (This is Kirsten borrowing Jessica's computer due to some technical problems!)

**Comment by kdorans** [comment]
http://onlinelibrary.wiley.com/doi/10.1002/ecs2.1242/full

**Comment by kdorans** [comment]
I found some resources that might help to explain a bit more about potential use of Bayesian compared with Frequentist approaches in Ecology: 
1. ResearchGate [discussion]( https://www.researchgate.net/post/How_to_approach_Bayesian_statistical_ecology_in_a_simple_way)
(Note: I do not think I agree with the text stated in one response, as I think Bayes can definitely be incorrect if assumptions used in the model are incorrect -
As to your final question, Mick McCarthy cites E.T. Jaynes: "... Bayesian and frequentist methods often generate numerically similar answers ... However, Bayesian methods have the distinct advantage that when the numerical results differ, the Bayesian methods are invariably correct (Jaynes, 1976)." )
2. I also found a paper that might be useful (uploaded [here](https://drive.google.com/open?id=0Bzm5SxWtJbpZN3k1eFFNQzl2S2c)).

**Comment by pvarelag** [comment]
Good practice in Bayesian network modelling. Environmental Modelling & Software
http://www.sciencedirect.com/science/article/pii/S1364815212001041
